packages/storefront-events-collector/src/contexts/requisitionList.ts

```diff
@@ -0,0 +1,26 @@
+import { RequisitionList } from "@adobe/magento-storefront-events-sdk/dist/types/types/schemas";
+
+import { RequisitionListContext } from "../types/contexts";
+
+const createContext = (requisitionList?: RequisitionList): RequisitionListContext | null => {
+    const mse = window.magentoStorefrontEvents;
+    const requisitionListCtx = requisitionList ?? mse.context.getRequisitionList();
+
+    if (!requisitionListCtx) {
+        return {
+            id: "",
+            name: "",
+            description: ""
+        }
```

Comment:
i think we should just return `{}` and not send event if these aren't present in the context

Reply:
This will have to return null or object with id, so going to send null.